src/main/resources/roleconfig/role_employment.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "name": "specific-access-approver-legal-operations",
+    "label": "Specific Access Approver Legal Operations",
+    "description": "Specific Access Approver Legal Operations role for staff users",
+    "category": "LEGAL_OPERATIONS",
+    "substantive": false,
+    "type": "ORGANISATION",
+    "patterns": [
+      {
+        "roleType": {
+          "mandatory": true,
+          "values": ["ORGANISATION"]
+        },
+        "grantType": {
+          "mandatory": true,
+          "values": ["STANDARD"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PUBLIC"]
+        },
+        "attributes": {
+          "jurisdiction": {
+            "mandatory": true,
+            "values": ["EMPLOYMENT"]
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "clerk",
```

Comment:
clerk is already in use by SSCS. So please remove this here and move SSCS one to common json file by adding SSCS and EMPLOYMENT jurisdictions

Reply:
addressed